tornjak-frontend/src/tables/entries-list-table.tsx

```diff
@@ -7,12 +7,36 @@ import {
     entriesListUpdateFunc
 } from 'redux/actions';
 import Table from './list-table';
+import { EntriesList } from "components/types";
+import { RootState } from "redux/reducers";
+import { DenormalizedRow } from "carbon-components-react";
 
 // EntriesListTable takes in 
 // listTableData: entries data to be rendered on table
 // returns entries data inside a carbon component table with specified functions
-class EntriesListTable extends React.Component {
-    constructor(props) {
+
+type EntriesListTableProp = {
+    // dispatches a payload for list of entries with their metadata info as an array of EntriesListType and has a return type of void
+    entriesListUpdateFunc: (globalEntriesList: EntriesList[]) => void,
+    // data provided to the entries table
+    data: {
+        key: string,
+        props: { entry: EntriesList }
+    }[] | string | JSX.Element[],
+    id: string,
+    // list of available entries as array of EntriesListType or can be undefined if no array present
+    globalEntriesList: EntriesList[] | undefined,
```

Comment:
is there a way we can make it so that it cant be undefined?

Reply:
Removed